nilearn/datasets/func.py

```diff
@@ -1799,6 +1802,39 @@ def fetch_surf_nki_enhanced(
     )
 
 
+def load_sample_surf_nki_enhanced(n_subjects=1):
```

Comment:
should probably provide `data_dir` to pass to `fetch_surf_nki_enhanced` or ship sample data with nilearn

Reply:
I think we don't want to ship much data with Nilearn.